README.md

```diff
@@ -203,8 +230,37 @@ To include all the files in a directory, use the `*` wild card (eg. `/var/log/*`
 
 Additionally, you can add the `--no_service` to omit the step of automatically creating `monasca-log-agent.service` in `/etc/systemd/system/`
 
-### Monasca-ui
+### Uninstalling Logs agent
```

Comment:
After uninstalling process, `monasca-log-agent` service isn't removed.
Is this the other bug?

Reply:
Did you checked before or after running `systemctl daemon-reload`?